doc/dev/development.md

```diff
@@ -143,11 +165,19 @@ git remote set-url --push upstream no_push
 
 ### Committing changes to your fork
 
-When you're happy with some changes, you can commit them and push them to your fork:
+When you're happy with some changes, you can commit them to your repo:
 
 ```shell
 git add .
 git commit
+```
+Then push the change to the fork. When prompted for authentication, using your
```

Comment:
You cannot join Istio without 2-factor being setup. With 2-factor, you have to push using token if you use HTTPS, which is what this flow is about.